### DISCUSSION TITLE
Main Ideas for ERPSAAS

### DISCUSSION THREAD

**Discussion by andrewdwallo** [andrewdwallo/erpsaas#2] [Ideas]
### Here is a general list of ideas I plan on integrating into ERPSAAS:

**1. Accounting Module:** 
- Accurate accounting based calculations for everything finance/accounting related
- Double Entry Accounting/Bookkeeping
- Financial Statements:
  Balance Sheet
  Income Statement
  Cash Flow Statements
  Statements of Shareholders' Equity
- General Journal
- General Ledger
- Trial Balance

**Integrate Plaid API for Accounting Module**
- Users/Companies will connect to their Bank Institution for Bank Feeds
- Companies will be able to get detailed reports of all transactions, statements, etc.
- Integration of Company Bank Institutions, Accounts, Transactions, Real-time Balances, Assets, Liabilities, Investments, Income, and more!
-  These accounts can either be assigned to the company as a whole or a specific department
- *More to use with this, just a general idea...

**Integration of Stripe API and Plaid Combined**
- Stripe API will be used for Companies to retrieve customer purchases along with invoices, tax statements, tax identification, payment methods/gateways, subscriptions, subscription trials, product purchases and checkouts, etc...
** This will be used along with plaid to generate financial statements for customer transactions/purchases

**Company Posts**
- Companies will be able to create posts that promote their company, any products/services they offer, etc.. that will be generated on the frontend for the public to see. 
- This will include product/services that the company wishes to list on the frontend to the public for customers to purchase using Plaid and Stripe API.
- Since companies are using Plaid and Stripe API,... after a customer purchases a product/service their financial statements, invoices, sales, and transaction statements will automatically be updated after customer purchase.

**These are the general ideas I have for this project, but there is much more I can think of!**
- Along with Accounting/Finance Module:

1. Procurement
2. Manufacturing
3. Inventory Management
4. Order Management
5. Warehouse Management
6. Supply Chain Management
7. Customer Relationship Management (CRM)
8. Project Management
9. Workforce Management
10. Human Resources Management (HR)
11. Ecommerce
12. Marketing Automation

**Comment by nnq1** [comment]
Thanks and great ideas

**Comment by pablozagni** [comment]
Hi. How can I help.

**Reply by KhairulAzmi21** [reply]
I want to help also !

**Reply by andrewdwallo** [reply]
Thank you all for your willingness to contribute and for your enthusiasm toward the project! At the moment, I'm deeply involved in preparing for a significant release, which encompasses a lot of complex logic that might be challenging to get up to speed with immediately.

However, your offer to help is greatly appreciated, and I'd love to involve you in the project's future development. After this next release, I plan to share details about upcoming features and areas where I could use your expertise and assistance. This will be a great opportunity for us to collaborate and bring fresh ideas to the table.

Please stay tuned for updates, and again, thank you for your support and interest in contributing!

**Comment by malukelereko** [comment]
hello am interested in helping in this project

**Comment by konstantinosbotonakis** [comment]
I believe I can help on Stripe, eCommerce, Product and Order Management.
